courses/views_tests.py

```diff
@@ -48,3 +54,92 @@ def test_doesnt_list_courses_from_unlive_programs(self):
         resp = self.client.get(reverse('course-list'))
 
         assert len(resp.json) == 0
+
+
+class ProgramEnrollmentTests(ESTestCase, APITestCase):
+    """Tests for the ProgramEnrollment API"""
+
+    @classmethod
+    def setUpTestData(cls):
+        super(ProgramEnrollmentTests, cls).setUpTestData()
+
+        cls.user1 = UserFactory.create()
+        cls.user2 = UserFactory.create()
+        cls.program1 = ProgramFactory.create(live=True)
+        cls.program2 = ProgramFactory.create(live=True)
+        cls.program3 = ProgramFactory.create(live=True)
+        for program in (cls.program1, cls.program2,):
+            ProgramEnrollmentFactory(
+                user=cls.user1,
+                program=program,
+            )
+        cls.url = reverse('user_program_enrollments')
+
+    def setUp(self):
+        super(ProgramEnrollmentTests, self).setUp()
+        for program in (self.program1, self.program2,):
+            ProgramEnrollmentFactory(
```

Comment:
Does this need to be run here and in `setUpTestData`? Wouldn't one of the two create a conflict?